tests/integration/targets/lookup_hashi_vault/runme.sh

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env bash
 
+# The default container does not seem to have virtualenv installed
+${ANSIBLE_TEST_PYTHON_INTERPRETER} -m pip install virtualenv
```

Comment:
Maybe add a link to https://github.com/ansible/ansible/issues/72738 in a comment ?